common/metrics/defs.go

```diff
@@ -2796,6 +2797,7 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		TaskWriteThrottlePerTaskQueueCounter:      NewRollupCounterDef("task_write_throttle_count_per_tl", "task_write_throttle_count"),
 		TaskWriteLatencyPerTaskQueue:              NewRollupTimerDef("task_write_latency_per_tl", "task_write_latency"),
 		TaskLagPerTaskQueueGauge:                  NewGaugeDef("task_lag_per_tl"),
+		NoRecentPollerTasksPerTaskQueueCounter:    NewRollupCounterDef("no_poller_tasks_per_tl", "no_poller_tasks"),
```

Comment:
1. Is a rollup useful here? It seems like this metric is sort of useless without the task queue name.
2. Do our rollup metrics even work right? I looked at the code and it's adding the same tags to the main one and the rollup, so how are they any different? @yiminc 
3. Even if they work, should we add new rollup metrics? Any decent monitoring system can already aggregate metrics by arbitrary sets of tags

Reply:
We need to revisit rollup metrics separately. We should not try to address rollup issue in this PR.